docker-compose.apps.yml

```
@@ -97,6 +96,36 @@ services:
+    deploy:
+      resources:
+        limits:
+          memory: 500m
+  mailhog:
+    image: mailhog/mailhog
+    container_name: mailhog
+    ports:
+      - "1025:1025"
+      - "8025:8025"
+    healthcheck:
+      test: [ "CMD", "wget", "--spider", "http://localhost:8025" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)